scripts/setup-e2e-tests.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# Helper AI - Playwright E2E Testing Setup Script
+# This script sets up everything needed for E2E testing
+
+set -e
+
+echo "🎭 Setting up Playwright E2E Testing for Helper AI"
+echo "=================================================="
+
+# Check if we're in the right directory
+if [ ! -f "package.json" ]; then
+    echo "❌ Error: Please run this script from the root of the Helper AI project"
+    exit 1
+fi
+
+echo "📦 Installing Playwright and dependencies..."
+pnpm install
+
+echo "🎭 Installing Playwright browsers..."
+pnpm exec playwright install --with-deps
+
+echo "🔧 Setting up test environment..."
+
+# Create .env.local if it doesn't exist
+if [ ! -f ".env.local" ]; then
+    echo "📝 Creating .env.local file..."
+    cat > .env.local << EOF
+# E2E Testing Environment Variables
+TEST_USER_EMAIL=test@example.com
+TEST_OTP=123456
+TEST_API_KEY=test-api-key
+
+# Add your other environment variables here
+EOF
+    echo "✅ Created .env.local - please update with your actual values"
+else
+    echo "✅ .env.local already exists"
+fi
+
+# Create test directories if they don't exist
+echo "📁 Creating test directories..."
+mkdir -p tests/e2e/.auth
+mkdir -p tests/e2e/screenshots
+mkdir -p playwright-report
+mkdir -p test-results
+
+echo "🔒 Setting up authentication state directory..."
+chmod 755 tests/e2e/.auth
+
+echo "🚀 Starting local services..."
+echo "   📊 Starting Supabase (if not already running)..."
+pnpm services:start > /dev/null 2>&1 || echo "   ⚠️  Services might already be running"
```

Comment:
`services:start` won't do anything if Supabase is already running anyway, so I don't think we need the rest?

```suggestion
pnpm services:start
```